tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -5088,18 +5172,25 @@ port::Status CudnnSupport::GetFusedConvolveRunners(
   auto cudnn = cudnn_->GetHandle(parent_, stream);
   auto op_graph_status = GetCudnnFusedOperationGraph(
       kind, input_type, bias_type, output_type, conv_scale, side_input_scale,
-      input_descriptor, filter_descriptor, bias_descriptor, output_descriptor,
-      convolution_descriptor, activation_mode, cudnn);
+      leakyrelu_alpha, input_descriptor, filter_descriptor, bias_descriptor,
+      output_descriptor, convolution_descriptor, activation_mode, cudnn);
   if (!op_graph_status.status().ok()) {
     return port::Status(port::error::INTERNAL,
                         absl::StrCat("Cudnn graph failed to build: ",
                                      op_graph_status.status().ToString()));
   }
   auto op_graph = std::move(op_graph_status).value();
 
+  bool need_side_input = true;
+  if (activation_mode == dnn::ActivationMode::kRelu6 ||
+      activation_mode == dnn::ActivationMode::kElu ||
+      activation_mode == dnn::ActivationMode::kLeakyRelu) {
+    need_side_input = false;
+  }
```

Comment:
I see that this logic is repeated in a few places (and is inverted, like we said).  This should be pulled out into a separate function?